CardView: describe scrolling and dragging API (#7491)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxCardView/1 Configuration/scrolling/useNative.md

---
id: dxCardView.Options.scrolling.useNative
type: Boolean | Enums.Mode
default: 'auto'
---
---
##### shortDescription
Specifies whether CardView should use native or simulated scrolling.

---

> This property can have one of the following values:

Reviewer: ```suggestion
This property accepts one of the following values:
```

<table class="dx-table">
    <tr>
        <th>scrolling.useNative</th>
        <th>Description</th>
    </tr>
    <tr>
        <td><i>"auto"</i></td>

>         <td>Native scrolling on all platforms, except non-Mac desktops.</td>

Reviewer 2: Minor
I know that this is copied from scrollable, but "except non-Mac desktops" is hard to read because of double negation
Related to same description in other places
